src/client/components/SiteHeader.tsx

```diff
@@ -132,6 +133,7 @@ const SiteHeader = ({ user }: Props) => {
       ) : "resource" in user ? (
         <React.Fragment>
           <SupportMenu />
+          <OrganizationDropdown organizations={user.resource.organizations} />
```

Comment:
We shouldn't show this if you haven't joined any organizations yet:
```suggestion
          {user.resource.organizations.length > 0 && (
            <OrganizationDropdown organizations={user.resource.organizations} />
          )}
```